pkgs/development/python-modules/genson/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, jsonschema
+}:
+
+buildPythonPackage rec {
+  pname = "genson";
+  version = "1.2.2";
+  format = "setuptools";
+
+  disabled = pythonOlder "3.7";
+
+  # The Pypi tarball doesn't contain some test data
+  src = fetchFromGitHub {
+    owner = "wolverdude";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-3RaGY/F//Zt3aOH5ZO+jvRc6vn0+yT14MTYg6yCzO3w=";
+  };
+
+  patches = [ ./fix-test-bin.patch ];
+
+  propagatedBuildInputs = [ jsonschema ];
+
```

Comment:
Since `python setup.py test` has been deprecated, it would be better to use `unittestCheckHook`.

see, https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/python.section.md#testing-python-packages-testing-python-packages

Reply:
Done